src/store/dynamodb/dynamodb-store.ts

```diff
@@ -20,18 +21,20 @@ export class DynamoDbStore implements ProductStore {
   private static ddbDocClient: DynamoDBDocumentClient =
```

Comment:
Have you thought about patching the client instead ([docs](https://awslabs.github.io/aws-lambda-powertools-typescript/latest/core/tracer/#patching-aws-sdk-clients))??

This way you could even potentially avoid to capture each method since they're only wrapping the SDK. By patching the client itself you'd get a subsegment for free in each action.

Reply:
About the method's capturing: you are correct in the sense that the logic doesn't do much other that wrapping the SDK client call, but I think it can bring value to developers to see how it's done. And it's more clear when you see it in the console.
<img width="1193" alt="Screenshot 2022-07-07 at 18 02 54" src="https://user-images.githubusercontent.com/47529391/177951168-7162045e-9ab5-4535-92ee-975102cec8d9.png">

Reply:
That being said, I will look into it :)

Reply:
No, you are right actually. The diff on the right of the screen was not showing me [this line](https://github.com/saragerion/serverless-typescript-demo/blob/feat/powertools/src/store/dynamodb/dynamodb-store.ts#L19) so I thought you were not capturing the client, but you are.

Ignore the comment please.